core/src/conn.rs

```diff
@@ -35,17 +38,20 @@ impl MongoConnection {
     /// The initial operation time if provided should come from and will take precedence over the
     /// setting specified in the uri if any.
     pub fn connect(
-        mut client_options: ClientOptions,
+        mut options: (ClientOptions, Option<ConnectionString>),
```

Comment:
```suggestion
        mut (options, cs): (ClientOptions, Option<ConnectionString>),
```
Should we name these in the arg list so we can refer to them more clearly below as `options` and `cs` instead of `options.0` and `options.1`, respectively?